smtlib-backends-cvc5/cbits/cvc5.cpp

```diff
@@ -91,10 +91,9 @@ char *cvc5_eval_smtlib2_string(cvc5_solver *slv, cvc5_parser *parser, char const
   {
     os << "(error \"" << e.getMessage() << "\")" << std::endl;
   }
-  // should catch (ParserException& e) here - see https://github.com/cvc5/cvc5/issues/10181
   catch (...) 
```

Comment:
No need to single out ParserException's then?

Reply:
They made the `CVC5ApiException` a superclass of `ParserException` now https://github.com/mudathirmahgoub/cvc5/blob/efac45b3a45b75c09625dfd879c588ac609313b1/include/cvc5/cvc5_parser.h#L275